primitives/chain-rococo/src/lib.rs

```diff
@@ -42,8 +43,26 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 };
 
 #[derive(parity_scale_codec::Encode, parity_scale_codec::Decode, Debug, PartialEq, Eq, Clone)]
+#[allow(non_camel_case_types)]
 pub enum Call {
-	MockPallet,
+	/// Westend bridge pallet.
+	///
+	/// Note this index has to be kept in-sync with Rococo runtime configuration:
+	/// https://github.com/paritytech/polkadot/blob/master/runtime/rococo/src/lib.rs#L272
```

Comment:
I don't think this points to what you want it to point to. 

I imagine you want this to point to the Westend bridge pallet instance of Rococo's `construct_runtime!` macro, but it's pointing elsewhere.

We may need to be more explicit in the comment, or tag it to a commit.

Reply:
Comment updated, please take a look.